src/components/Tooltip/Tooltip/index.scss

```diff
@@ -127,6 +127,10 @@
     .tether-element {
       z-index: $z_index_popovers;
 
+      &.underModal {
```

Comment:
Where is this class used?

Reply:
In another repo. This coment will be obsolete :trollface:

Reply:
:trollface: